grantnav/frontend/templates/currency_stats.html

```diff
@@ -0,0 +1,25 @@
+{% load frontend %}
+
+<dl class="dl-horizontal top-stats-search">
+  {% with currency=results.aggregations.currency_stats.buckets|first|get:"key"|upper buckets=results.aggregations.currency_stats.buckets%}
+    <dt>Total {{currency}} grants</dt> <dd>{{buckets|first|get:"amount_stats"|get:"count"|get_amount}}</dd>
+    <dt>Total {{currency}} awarded</dt> <dd>{{currency|currency_symbol}}{{buckets|first|get:"amount_stats"|get:"sum"|get_amount}}</dd>
+    <dt>Largest {{currency}} award</dt> <dd>{{currency|currency_symbol}}{{buckets|first|get:"amount_stats"|get:"max"|get_amount}}</dd>
+    <dt>Smallest {{currency}} award</dt> <dd>{{currency|currency_symbol}}{{buckets|first|get:"amount_stats"|get:"min"|get_amount}}</dd>
```

Comment:
Given that ```get``` filter defaults to ```None```, I think these chains of ```get```s might in principle give a 500 (_" 'NoneType' object has no attribute 'get' "_)

Reply:
If this was data that we got from a user then I would agree.  However this data is returned from elasticsearch so I believe we can rely on the structure being the same and all keys being there.

Reply:
:+1: I thought about that too, but trying to access a property of a ```None``` type is such a common error that I thought it was worth mentioning